packages/dev/core/src/Misc/deepCopier.ts

```diff
@@ -15,6 +15,8 @@ const CloneValue = (source: any, destinationObject: any) => {
         return source.clone();
     } else if (Array.isArray(source)) {
         return source.slice();
+    } else if (typeof source === "object") {
+        return { ...source };

```

Comment:
if I remember correctly, last time we added that @deltakosh commented that this is not needed. I can't find the conversation, so if he can remind me (or contradict me :-) ) it would be great.

Reply:
If I can remember correctly we were discussing lines 16-17:

```javascript
    } else if (Array.isArray(source)) {
        return source.slice();
```

Reply:
All good here then. This is, however, not a deep copy. It is a shallow copy.

Reply:
So can we resolve this conversation please?

Reply:
if shallow copy it enough, sure